### PR TITLE
data: Zaid hassan

### PR DESCRIPTION
**Pull request by zaid-hassan**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8567"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @zaid-hassan, 

You'd merge conflicts due to an outdated branch. Please update your fork branch to resolve conflicts! 

If you need any help, you can checkout: https://youtu.be/eze7Y4Rxwtc

File: data/zaid-hassan.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Zaid Hassan",
+  "type": "personal",
```
Comment (SaraJaoude) [inline data/zaid-hassan.json:3]: this is no longer required:
```suggestion
```

**Review by Sindhuinti — APPROVED** [review]

**Comment by SaraJaoude** [comment]
@Sindhuinti please do not approve changes where there are comments which have not been addressed. This is confusing for the author and also the maintainers.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the Docs which will help with the suggested changes the maintainers have proposed: https://linkfree.io/docs

**Pull request closed without merging**